Procfile

```diff
@@ -1 +1,2 @@
+release: bundle exec rake db:migrate
```

Comment:
This is called "all the time" on review apps as well as on the dev deployment. As per [the docs](https://devcenter.heroku.com/articles/release-phase#when-does-the-release-command-run), this is called after:

- An app build
- A config var set by a user
- A pipeline promotion
- A rollback
- A release via the platform API

I figured it would make sense to run migrations so the dev is always up to date.